src/sass/_common/common.scss

```diff
@@ -1,1130 +1,1315 @@
 /* DO NOT include any rules for specific pages in this file. */
 /* theme elements */
 
-html, body {
```

Comment:
I think your linter accidentally differently formatted this CSS file, I believe this should somehow be reverted, otherwise I believe it can cause conflicts for other developers.